librz/search/options.c

```diff
@@ -66,6 +66,12 @@ RZ_API bool rz_search_opt_set_max_threads(RZ_NONNULL RzSearchOpt *opt, RzThreadN
 	return true;
 }
 
+RZ_API bool rz_search_opt_set_show_progress(RZ_NONNULL RzSearchOpt *opt, bool show_progress) {
+	rz_return_val_if_fail(opt, false);
+	opt->show_progress = show_progress;
```

Comment:
I think though this should be an extra option. Something like `show_progress_verbose`. Because if the file is big, there will be many intervals, which clutters the output. Maybe it could be also a choice:

`number_of_hits`, `percent`, hits_per_interval/sub-graph/knowledge domain etc` etc. This might also we useful for later, when we have searches over graphs or the KB.

Reply:
I think that a hybrid approach with `show_progress` i.e. it accepts `false`, `true` and the special value `interval_hits` (shorter than `hits_per_interval`) is currently the best way to handle this, but it will take some time.

Reply:
See 97ae8557d1c660f11744f8b81f1558b94fb9479f ... I'm using `intervals` since it's shorter than `interval_hits` yet basically has the same meaning (i think)